### PR TITLE
bump flow-go version

### PR DESCRIPTION
**Pull request by ramtinms**
follow up to #174, this pr updates flow-go version to match with the latest on the flow-go master
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to GitHub issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Review by sideninja — APPROVED** [review]

**Review by janezpodhostnik — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -13,9 +13,9 @@ require (
 	github.com/improbable-eng/grpc-web v0.12.0
 	github.com/logrusorgru/aurora v2.0.3+incompatible
 	github.com/onflow/cadence v0.24.2-0.20220627202951-5a06fec82b4a
-	github.com/onflow/flow-go v0.25.13-0.20220708171206-7015f054a69f
+	github.com/onflow/flow-go v0.26.14-test-synchronization.0.20220715235626-9652b94c903a
```
[the remark this answers is not in the document]
Comment (sideninja) [reply go.mod:16]: Yes, I agree. I oversaw that sorry.

[the remark this answers is not in the document]
Comment (ramtinms) [reply go.mod:16]: we don't have a good way to tag pre-released tags on flow-go side, let me chat with the team to see what is the plan there.

[the remark this answers is not in the document]
Comment (ramtinms) [reply go.mod:16]: @Kay-Zee thoughts on the pre-release versioning?

[the remark this answers is not in the document]
Comment (sideninja) [reply go.mod:16]: I don't think it's a big deal if we do this if needed but I don't want to become a habit. Cadence is also being targeted with a commit lately.

[the remark this answers is not in the document]
Comment (SupunS) [reply go.mod:16]: Any verdict here? I also need to have cadence + flow-go versions bumped in the emulator (https://github.com/onflow/flow-emulator/pull/180) , which is needed for the [cadence test framework](https://github.com/onflow/cadence/pull/1827)

[the remark this answers is not in the document]
Comment (ramtinms) [reply go.mod]: I'm trying to update v0.27 branch and tag a pre-release for this PR

[the remark this answers is not in the document]
Comment (ramtinms) [reply go.mod]: I started a new set of tags for pre-release (e.g. `v0.27.0-pre-release-1` for this one) so we could do tagging for pre-release

[the remark this answers is not in the document]
Comment (SupunS) [reply go.mod]: yes, the changes I needed are in flow-go master

**Pull request merged**